gtfs.yml

```diff
@@ -276,6 +276,11 @@
       text: Funicular
     columnWidth: 12
     helpContent: The route_type field describes the type of transportation used on a route. Valid values for this field are...
+  - name: route_sort_order
```

Comment:
In my [other comment](https://github.com/catalogueglobal/datatools-ui/pull/264#issuecomment-442922044) I had mentioned having this field be displayed next to route_type and changing route_type's width to 6. I was also implying that route_sort_order should have a width of 6. I now realize my language wasn't super clear. Could you make that fix as well? Or if you already tried it and it looked weird, no worries.

Reply:
Ah, I see what you mean now.  They can fit side-by-side.  The updated `gtfs.yml` should have the desired changes.